src/System.Text.Json/src/System/Text/Json/Reader/JsonReaderHelper.Unescaping.cs

```diff
@@ -107,6 +132,46 @@ public static bool UnescapeAndCompare(ReadOnlySequence<byte> utf8Source, ReadOnl
             return result;
         }
 
+        public static bool TryDecoderInPlaceHelper(Span<byte> utf8Unescaped, out byte[] bytes)
```

Comment:
Why is there an `r` in Decode?  Decoder (with the r) is a noun, so it'd be a type or property name, or after a verb in a method.  Here it's used as a verb, so I think you want TryDecode (no `r`).  Going further, and expanding on @gfoidl's comment, `TryDecodeBase64InPlace` seems a good name.